lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -25,7 +25,7 @@ def as_json
           meta = meta_for(association)
           hash[:meta] = meta if meta
```

Comment:
Now for code review. :-)

I don't like that ensure_valid! just adds a blank meta. The method name is misleading.
Since the logic is so simple, can you add that logic to this method?